cvat/apps/engine/serializers.py

```diff
@@ -3137,6 +3139,7 @@ def update(self, instance: models.Job, validated_data: dict[str, Any]) -> models
             db_data.deleted_frames = updated_deleted_task_frames
             db_data.save(update_fields=['deleted_frames'])
 
+        instance.touch()
```

Comment:
Hmm, I don't think this is correct. This code only makes changes to the `Data` object, which logically belongs to the task rather than the job. So I don't think it makes sense to touch the job here.